reinforcement_learning/common/sagemaker_rl/tf_serving_utils.py

```diff
@@ -17,8 +17,8 @@ def change_permissions_recursive(path, mode):
     for root, dirs, files in os.walk(path, topdown=False):
         for dir in [os.path.join(root, d) for d in dirs]:
             os.chmod(dir, mode)
-    for file in [os.path.join(root, f) for f in files]:
-        os.chmod(file, mode)
+        for file in [os.path.join(root, f) for f in files]:
```

Comment:
what is this change, this seems changing the logic?

Reply:
This is an existing bug since the launch of SageMaker RL. There was plan to fix it but got delayed.